### ISSUE TITLE
BUG: VidCoder logs stored in wrong directory (Roaming vs Local)

### ISSUE THREAD

**Issue by bcurran3** [RandomEngy/VidCoder#450]
VidCoder logs are stored in AppData\Roaming.

The Roaming directory is for files that should transfer to another computer when using a roaming profile; i.e. important configuration settings/user preferences.

The log files should be stored in AppData\Local since they are computer specific and have no need to roam with the user account.

Not necessarily the authority, but a good explanation and first to show up in a Google search:
REF: https://www.howtogeek.com/318177/what-is-the-appdata-folder-in-windows/

**Comment by RandomEngy** [comment]
That makes sense. Is this causing a problem for you?

**Comment by bcurran3** [comment]
No. My account is in a domain but not currently configured with a roaming profile.

Priority: Low
